src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
Here u can create separate variable for ternary logic for ex. const isCompleted = todo.completed ? 'TodoInfo--completed' : ''}; for better readability =)